test/test_integration_credentials_netrc.py

```diff
@@ -15,20 +15,19 @@
 from conjur.data_object import CredentialsData
```

Comment:
first docstring of line 4 reads: CLI Integration Credentials tests

This should be:

CLI Integration Credentials using .netrc tests

This test file handles the login/logout test flows when running
`conjur login`/`conjur logout` and writing to .netrc